src/styles.css

```diff
@@ -84,19 +84,27 @@ nav {
 }
 
 main {
-  padding: 16px;
+  /*  padding: 16px; */
```

Comment:
unless really necessary, please don't commit commented-out code to the main branch.